Gemfile

```diff
@@ -35,6 +35,7 @@ group :development do
   gem "rubocop-rspec", '= 2.0.1',                      require: false
   gem "rb-readline", '= 0.5.5',                        require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "github_changelog_generator", '~> 1.15',         require: false if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')
+  gem "webmock",                                       require: false
```

Comment:
gem should probably be added to .sync.yml too to make sure it survives 'pdk update' command.

Reply:
.sync.yml file updated